.eslintrc

```diff
@@ -88,6 +88,39 @@
           }
         ]
       }
+    },
+    {
+      "files": [
+        "public/app/plugins/datasource/testdata/*.{ts,tsx}",
+        "public/app/plugins/datasource/testdata/**/*.{ts,tsx}"
+      ],
+      "rules": {
+        "no-restricted-imports": [
+          "error",
+          {
+            "paths": [
+              // generic rules
```

Comment:
unfortunately, I cannot reuse the options defined for this rule from line 24 so I had to duplicate them